packages/@okta/vuepress-site/docs/reference/okta-expression-language/index.md

```diff
@@ -400,6 +400,14 @@ In [API Access Management](/docs/concepts/api-access-management/) custom authori
 To include a granted scope array and convert it to a space-delimited string, use the following expression:<br>
 `String.replace(Arrays.toCsvString(access.scope),","," ")`
 
+## Expressions in group rules
+
+The set of expressions allowed in group rules is limited to `String`, `Arrays`, and `user` expressions.
```

Comment:
I think we can be a bit more concise. May I suggest:
```suggestion
Group rules only allow `string`, `array`, and `user` expressions.
```
Also, I'm pretty sure these expression types don't need to be capitalized. We should have Susan Harper review this PR, too. She's our copyeditor and style guide guru.

Reply:
Updated - let me know what you think.

Reply:
I forgot to mention that `String` and `Arrays` refer to product functionality. Someone has to type in the capitalized version in group rules and can't use the lowercase version.